q2_assembly/quast/quast.py

```diff
@@ -295,6 +312,13 @@ def _visualize_quast(
         # Copy results to output dir
         copy_tree(results_dir, os.path.join(output_dir, "quast_data"))
 
+        # Save the downloaded references
+        if not references:
+            download_references = os.path.join(
+                results_dir, "quast_downloaded_references"
+            )
+            _move_references(download_references, output_dir)
+
```

Comment:
What would happen here if there was no internet connection _and_ the user did not provide references?